source/src/protocols/ligand_docking/HighResDocker.cc

```diff
@@ -282,11 +291,26 @@ HighResDocker::apply(core::pose::Pose & pose) {
 	score_fxn_->score( pose ); // without this neither of the movers below were working
 	// I believe that this may have been related to adding constraints incorrectly at other places in my code.
 	// Rigid body exploration
-	utility::vector1<protocols::moves::MoverOP> rigid_body_movers= create_rigid_body_movers(pose);
+	utility::vector1<protocols::moves::MoverOP> rigid_body_movers = create_rigid_body_movers(pose);
+
+    core::Size ligand_index = 0;
+    for ( core::Size i(1), imax(pose.total_residue()); i<=imax; ++i ) {
+        core::conformation::Residue const & residue( pose.residue(i) );
+        if ( residue.is_ligand() ) {
+            ligand_index = i;
+        }
+    }
+    core::conformation::Residue original_ligand(pose.residue(ligand_index));
```

Comment:
Prefer using a ResidueOP and `.clone()` to make a copy. I don't think it makes a difference here, but it does in other contexts where there's multiple derived types, it plays better with subtypes (the approach here would lose subtype info.)